json-rpc/types/src/views.rs

```diff
@@ -464,6 +464,18 @@ pub struct TransactionView {
     pub vm_status: VMStatusView,
     pub gas_used: u64,
 }
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+pub struct TransactionsWithProofsView {
+    pub first_transaction_version: Option<u64>,
+    pub transactions_blobs: Vec<BytesView>,
```

Comment:
what's a transaction blob? My guess is that it's an LCS serialized transaction, in this case why not `serialized_transactions`?

Reply:
(or just transactions)

Reply:
It is, I just used the same terminology as for AccountStateWithProofView. I can rename it.

Reply:
but in the case of account state, the term account blob is the technical term we use : o